internal/test/wmcb/wmcb_test.go

```diff
@@ -182,6 +236,151 @@ func (vm *wmcbVM) remoteDownloadExtract(url, sha, remoteDownloadFile, remoteExtr
+func (vm *wmcbVM) initializeTestConfigureCNIFiles(ovnHostSubnet string) error {
+	// Create the CNi directory C:\Windows\Temp\cni on the Windows VM
+	_, stderr, err := vm.Run("if not exist "+winCNIDir+" mkdir "+winCNIDir, false)
+	if err != nil {
+		return fmt.Errorf("unable to create remote directory %s: %v\n%s", remoteDir, err, stderr)
+	}
+
+	// Download and extract the CNI binaries on the Windows VM
+	err = vm.remoteDownloadExtract(cniPlugins, remoteDir+"cni-plugins-windows-amd64-v0.8.2.tgz", winCNIDir)
+	if err != nil {
+		return fmt.Errorf("unable to download CNI package: %v", err)
+	}
+
+	// Create the CNI config file locally
+	cniConfigPath, err := createCNIConf(ovnHostSubnet)
+	if err != nil {
+		return fmt.Errorf("error creating local cni.conf: %v", err)
+	}
+
+	// Copy the created config to C:\Window\Temp\cni\config\cni.conf on the Windows VM
+	err = vm.CopyFile(cniConfigPath, winCNIConfigPath)
+	if err != nil {
+		return fmt.Errorf("error copying %s --> VM %s: %v", cniConfigPath, winCNIConfigPath, err)
+	}
+	return nil
+}
+
+// initializeHybridOverlayBinary creates the files on the Windows node needed for running "configure-cni"
+func (vm *wmcbVM) initializeHybridOverlayBinary() error {
+	hybridOverlayURL, err := framework.GetReleaseArtifactURL(hybridOverlayName)
+	if err != nil {
+		return fmt.Errorf("unable to get %s URL: %v", hybridOverlayName, err)
+	}
+
+	hybridOverlaySHA, err := framework.GetReleaseArtifactSHA(hybridOverlayName)
+	if err != nil {
+		return fmt.Errorf("unable to get %s SHA: %v", hybridOverlayName, err)
+	}
+
+	hybridOverlay := pkgInfo{
+		url:     hybridOverlayURL,
+		sha:     hybridOverlaySHA,
+		shaType: "sha256",
+	}
+
+	err = vm.remoteDownload(hybridOverlay, hybridOverlayExecutable)
+	if err != nil {
+		return fmt.Errorf("unable to download %s on VM: %s", hybridOverlayURL, err)
+	}
+	return nil
+}
+
+// handleHybridOverlay ensures that the hybrid overlay is running on the node
+func (vm *wmcbVM) handleHybridOverlay(nodeName string) error {
+	// Check if the hybrid-overlay is running
+	_, stderr, err := vm.Run("Get-Process -Name \"hybrid-overlay\"", true)
+
+	// stderr being empty implies that an hybrid-overlay was running. This is to help with local development.
+	if err == nil || stderr == "" {
+		return nil
+	}
+
+	// Wait until the node object has the hybrid overlay subnet annotation. Otherwise the hybrid-overlay will fail to
+	// start
+	if err = waitForHybridOverlayAnnotation(nodeName); err != nil {
+		return fmt.Errorf("error waiting for hybrid overlay node annotation: %v", err)
+	}
+
+	// Start the hybrid-overlay in the background over ssh. We cannot use vm.Run() and by extension WinRM.Run() here as
+	// we observed WinRM.Run() returning before the commands completes execution. The reason for that is unclear and
+	// requires further investigation.
+	go vm.RunOverSsh(hybridOverlayExecutable+" --node "+nodeName+
+		" --k8s-kubeconfig c:\\k\\kubeconfig > C:\\k\\hybrid-overlay.log 2>&1", false)
```

Comment:
This should write to the c:\k\log directory so we can capture it, and since its nicer to have all the logs in one place (in my opinion :smile:)